round.pg

```diff
@@ -0,0 +1,63 @@
+with args as (select
```

Comment:
This seems like an accidental commit?

Reply:
I just wanted to keep it until the PR was done. Do we have a private repository for things we want to keep around but shouldn't be in the source tree?

Reply:
Just as a backup; it took a while to write it. But I can keep it local, that's fine.